orchagent/portsorch.cpp

```diff
@@ -2777,6 +2829,13 @@ void PortsOrch::doVlanTask(Consumer &consumer)
                         gIntfsOrch->setRouterIntfsMac(vl);
                     }
                 }
+                if (!hostif_name.empty())
+                {
+                    if (!createVlanHostIntf(vl, hostif_name))
+                    {
+                        throw runtime_error("Cannot create VLAN host interface");
```

Comment:
Lets handle the failure gracefully as this is for monitoring Vlan. We can erase` it` and continue.

Reply:
Changed.